canonical/ubuntu.com
# /k8s/compare copy update

## Done

- Updated table heading as per [doc](https://docs.google.com/document/d/1JngYmRhohPo62yU72V1xpzCCILI21SmHKKttBxHjPog/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13583.demos.haus/kubernetes/compare
- See that the requested change was made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8844
